pkg/events/events.go

```diff
@@ -0,0 +1,50 @@
+package events
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/tools/record"
+)
+
+// Event message format "medik8s <operator shortname> <message>"
+const customFmt = "[remediation] %s"
```

Comment:
any reason to not use this? 🤔

Reply:
none indeed, it was intended to be used :shrug: 
I'll fix this